setup.cfg

```diff
@@ -4,3 +4,7 @@ with-coverage=1
 cover-package=baseplate
 cover-html=1
 cover-html-dir=build/coverage
+
+[flake8]
+ignore = E501,E128,E226
```

Comment:
E226 actually doesn't jive with the real PEP-8 doc. The actual rule is 

> Always surround these binary operators with a single space on either side: assignment ( = ), augmented assignment ( += , -= etc.), comparisons ( == , < , > , != , <> , <= , >= , in , not in , is , is not ), Booleans ( and , or , not ).

while E226 enforces for all operators. This shows up in baseplate for some math like https://github.com/reddit/baseplate/blob/master/baseplate/core.py#L103